### PR TITLE
Adding basic test for Cryptographic functions

### LINKED ISSUE #1659: support for Web Crypto API
It doesn't seem like Modernizr currently supports checks for the Web Crypto API. Both Chrome and Firefox (haven't checked others) have an implementation in their latest browsers under `window.crypto.subtle`.

- https://developer.mozilla.org/en-US/docs/Web/API/Web_Crypto_API
- https://developer.mozilla.org/en-US/docs/Web/API/SubtleCrypto
- http://caniuse.com/#feat=cryptography

Is there a plan to add support for this API soon?

### PR DESCRIPTION
**Pull request by roblarsen**
Closes #1659

### CODE REVIEW

**Comment by ryanseddon** [comment]
Looks like you've got some liniting errors

File: feature-detects/crypto/cryptography.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  ]
+}
+!*/
+/* DOC
+Detects support for the cryptographic functionality available under window.crypto.subtle
+*/
+define(['Modernizr', 'prefixed', 'is'], function(Modernizr, prefixed, is) {
+  var crypto = prefixed('crypto', window);
+  var hasSubtle = false;
+  if ("subtle" in crypto) {
```
Comment (ryanseddon) [inline feature-detects/crypto/cryptography.js]: New line above here

File: feature-detects/crypto/cryptography.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    hasSubtle = true;
+  }
+
```
Comment (ryanseddon) [inline feature-detects/crypto/cryptography.js]: Remove so only one new line

File: feature-detects/crypto/cryptography.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+define(['Modernizr', 'prefixed', 'is'], function(Modernizr, prefixed, is) {
+  var crypto = prefixed('crypto', window);
+  var hasSubtle = false;
+  if ("subtle" in crypto) {
+    hasSubtle = true;
+  }
+  
+
+  Modernizr.addTest('cryptography', hasSubtle);
+});
```
Comment (ryanseddon) [inline feature-detects/crypto/cryptography.js]: Make sure you have an eof

**Comment by patrickkettner** [comment]
@roblarsen what browsers were you able to test in?

**Comment by roblarsen** [comment]
@patrickkettner What I have locally. I'll spin up some screen shots and compare with caniuse...

**Comment by roblarsen** [comment]
With the above fix:
 https://www.browserstack.com/screenshots/2bd341cf7a590ec047bab12bc7a7757b2f170567

I think this is as expected across that whole pile of browsers

**Comment by patrickkettner** [comment]
just a few nits

1. following existing convention, this should be `feature-detects/crypto.js`
2. I think that the detect name should be `crypto`
3. squish down to a single commit plzandthnx

**Pull request merged**